Source/AkiSupport/Singleplayer/Patches/Progression/MidRaidQuestChangePatch.cs

```diff
@@ -25,24 +25,28 @@ protected override MethodBase GetTargetMethod()
         [PatchPostfix]
         private static void PatchPostfix()
         {
+            Logger.LogDebug($"[MidRaidQuestChangePatch] PatchPostfix");
             var gameWorld = Singleton<GameWorld>.Instance;
-            if (gameWorld != null)
+            if (gameWorld == null)
             {
                 Logger.LogError($"[MidRaidQuestChangePatch] gameWorld instance was null");
 
                 return;
             }
 
             var player = gameWorld.MainPlayer;
-            Logger.LogDebug($"[MidRaidQuestChangePatch] PatchPostfix");
 
             var questController = (QuestController)ReflectionHelpers.GetFieldFromType(player.GetType(), "_questController").GetValue(player);
-            if (questController != null)
+            if (questController == null) 
+            {
+				Logger.LogError($"[MidRaidQuestChangePatch] questController instance was null");
+
+				return;
+			}
```

Comment:
Not sure what's going on with the formatting here.  It looks normal inside Visual Studio for me locally.